.github/workflows/ci.js.yml

```diff
@@ -10,26 +10,15 @@ on:
     branches: ["master"]
 
 jobs:
-  build:
+  run-ci:
     runs-on: ubuntu-latest
-
-    strategy:
-      matrix:
-        node-version: [16.x]
-        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
-
     steps:
       - uses: actions/checkout@v3
-      - name: Use Node.js ${{ matrix.node-version }}
+      - name: Use Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: ${{ matrix.node-version }}
-          cache: "npm"
```

Comment:
why removed (and didn't used yarn cahe https://github.com/actions/setup-node#caching-global-packages-data)?

Reply:
fixed